apps/condo/domains/miniapp/schema/B2CAppMessageSetting.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * Generated by `createschema miniapp.B2CAppMessageSettings 'app?:Relationship:B2CApp:CASCADE; blockReason:Text; blockedNotificationTypes:Text; throttlingTime:Integer;'`
+ */
+
+const { historical, versioned, uuided, tracked, softDeleted, dvAndSender } = require('@open-condo/keystone/plugins')
+const { GQLListSchema } = require('@open-condo/keystone/schema')
+
+const access = require('@condo/domains/miniapp/access/B2CAppMessageSetting')
+const { getMessageTypeField } = require('@condo/domains/notification/schema/fields/MessageType')
+
+
+const B2CAppMessageSetting = new GQLListSchema('B2CAppMessageSetting', {
```

Comment:
This model is not really differ from the old one. Why don't you just add a new fields to the `MessageAppBlackList`?